Dockerfile

```diff
@@ -13,6 +13,20 @@ COPY . ./
 ARG VERSION
 RUN go install -ldflags "-X github.com/sourcegraph/zoekt.Version=$VERSION" ./cmd/...
 
+FROM rust:alpine3.17 AS rust-builder
+
+RUN apk update --no-cache && apk upgrade --no-cache && \
+    apk add --no-cache git musl-dev>=1.1.24-r10 build-base
+
+RUN git clone https://github.com/sourcegraph/sourcegraph && cd sourcegraph && git reset --hard 6dd16ddde8a02f3bf3fe36165e9724727277d97a && cd /
```

Comment:
nit: rather just fetch the archive tarball. much faster than cloning. in this case it would be something like https://github.com/sourcegraph/sourcegraph/archive/6dd16ddde8a02f3bf3fe36165e9724727277d97a.tar.gz

We don't do exactly the same thing, but you can take some inspiration from our install-ctags-alpine.sh script for how to structure a dependency like this. Side-note: it looks like that hasn't been bumped in a long time. Just pointing that out incase we have made ctags specific improvemnts.

Reply:
oh good call wrt the tarball. I think we haven't changed anything substantial in universal-ctags since then :)